doc/user/mpi-io.md

```diff
@@ -83,9 +112,14 @@ To run an example:
 3. At the client side, the following environment variables need to be set:
    `export DAOS_POOL=puuid; export DAOS_SVCL=svcl; export DAOS_CONT=cuuid`.
    Alternatively, the unified namespace mode can be used instead.
-3. Run the client application or test.
+3. Run the client application or test. 
+   MPI-IO applications should work seamlessly by just prepending `daos:` 
+   to the filename/path to use the DAOS ADIO driver.
+
+
+## Known limitations
 
-Limitations to the current implementation include:
+Limitations of the current implementation include:
 
 -   Reading Holes does not return 0, but leaves the buffer untouched.
```

Comment:
@mchaarawi Is this limitation still correct? I think that we have addressed that.

Reply:
yes this has been addressed. i will update this later this week